library/core/crash-info/src/main/java/org/quiltmc/qsl/crash/mixin/CrashReportSectionMixin.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2021 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.crash.mixin;
+
+import net.minecraft.block.BlockState;
+import net.minecraft.util.crash.CrashReportSection;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.HeightLimitView;
+import org.quiltmc.qsl.crash.api.CrashReportEvents;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+import org.spongepowered.asm.mixin.injection.callback.LocalCapture;
+
+@Mixin(CrashReportSection.class)
+public abstract class CrashReportSectionMixin {
+	@Inject(method = "addBlockInfo", at = @At("TAIL"), locals = LocalCapture.CAPTURE_FAILHARD)
```

Comment:
This doesn't need to capture any locals.
```suggestion
	@Inject(method = "addBlockInfo", at = @At("TAIL"))
```